chapters/14-team-package-dev.Rmd

```diff
@@ -1,14 +1,93 @@
+interacting with those in your team can help build a healthy and productive team
+environment.
+
+## Overview (remove this header later)
+
+- Making use of GitHub Organizations/Teams, member roles
+    - Text: 
+        - Centralize team activity, more settings to control how team members
+        contribute, more control for other members
+        - Different roles and permissions of members ("owners", "maintainers", etc)
+    - Exercise:
+        - Solo: Create a new GitHub Organization and make a new (random?) repo 
+        in the newly created Organization. (or move the package repo over into the
+        Organization?)
+        - Team: 
+            - Discuss within your team who wants to be assigned the "owner" of
+            the organization
+            - One of the owners create a new GitHub Organization
+            - Add the other team members and set their permission level to maintainer,
+            (and owner if there will be more than one.)
+            - Create new repo of the ... (FIXME: not sure which repo should be created)
+- Setting up two-factor authentication
+    - Text: What is it and why to use it.
+    - Text: In general 2FA is highly highly encouraged, but on team-based projects,
+    its even more so important, especially for repo owners and organization "owners".
+    - Exercise: Set up 2FA for your GitHub account.
+- Branch protection
+    - Text: Used to assign a specific person or people to be responsible for specific
+    branches, to provide a level of accountability and prevent unreviewed content
+    from being added to package.
+    - Text: Generally, depending on skill of member in git/github, best give
+    more team members access to modifying branch than to have less. But ultimately
+    it is a safety and accountability feature that needs to be discussed with team.
+    - Exercise: 
+        - Solo: In the Organization repo, set up branch protection for the main branch.
+        - Team:
+            - In the organization repo, set up branch protection for the main branch
+            and assign the owners as the only ones who can push to the branch.
+            FIXME: Make sure this is the correct terminology.
+- Relying on and using code of conduct, contributing guide, governance to build
+culture and standards
+    - Text: An implicit team culture will naturally evolve as people work together,
+    but having purely implicit cultural norms makes it difficult for newcomers
+    and for dealing with problems that come from misunderstanding of these cultural
+    practices. Much of this comes from the code of conduct and contributing guidelines,
+    but there are other aspects that can be team-, country-, etc specific.
+    - Exercise:
+        - (Solo/team?): Brainstorm come explicit expected behaviours, norms, 
+        practices that you would like in a team environment. Write this text
+        down and put it in the (FIXME: where? governance? conduct?)
```

Comment:
I believe the code of conduct is first created in [chapter 10](https://github.com/merely-useful/r-rse/blob/main/chapters/10-community-involvement.Rmd) where the focus is on package users and arms-length community contributors. It seems like the code of conduct info described here should go in a separate file: one idea might be to create a separate admin repo in the organization for meetings and have this go in that repo's CODE-OF-CONDUCT.md. At least anecdotally, that seems like what collaborating groups of people do (i.e. us and UofT Coders) :)